app/core/components/collector/test/test_proper_motion.py

```diff
@@ -15,7 +15,7 @@ def time():
 
 @pytest.fixture
 def target():
-    return SiderealTarget('', set(), TargetType.OTHER,
+    return SiderealTarget('', frozenset(), TargetType.OTHER,
```

Comment:
We should be using `frozenset` whenever possible, especially for something like magnitudes because they are not going to change. Is there a reason why you see that we should use `set` over `frozenset`?

These objects should be as immutable as possible, IMO, unless we have a very good reason for them to not be.

Reply:
I agree but frozentset was causing issues I recall? or I'm just getting confused with some other part of the code that we do `frozenset` to `set`?